app/guide/how-to-publish.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: How to publish a design history entry
+---
+
+Publishing a design history entry currently requires a bit of familiarity with [GitHub](https://github.com) and [Markdown](https://www.markdownguide.org), but we can help you if needed.
+
+To contribute you will need a free GitHub account, and for your account to be added to this project.
+
+Once this is done, you should follow these steps:
+
+1. Create a branch. The name of the branch doesn’t matter too much but you could name it after your post title.
+2. Add a new markdown file in `app/posts/[service-name]/YYYY/MM/[post-name].md`. It's often easier to duplicate an existing post as that will have the headers you need.
```

Comment:
```suggestion
2. Add a new markdown file in `app/posts/[service-name]/YYYY/MM/[post-name].md`. 

It’s often easier to duplicate an existing post. That way, you will have the headers that contain basic information that all posts need.
```

Reply:
I think this would break the numbered list and cause it to restart from 1?

Could not use a numbered list at all, and instead just have paragraphs?

Reply:
or just one paragraph